cime_config/usermods_dirs/NEON/FATES/MLBS/shell_commands

```diff
@@ -2,9 +2,13 @@
 ./xmlchange NEONSITE=MLBS
 ./xmlchange PTS_LON=279.47575
 ./xmlchange PTS_LAT=37.37783
-./xmlchange DATM_YR_END=2019
-# Different default number of months to run for transient case
-if  [[ $compset =~ ^HIST ]]; then
-  ./xmlchange DATM_YR_END=2020
-  ./xmlchange STOP_N=24
+# NEON precipitation data for this site is bad so end early
```

Comment:
@TeaganKing that's a nice insight. "Bad" is a value judgement which is maybe not what we want here. We should make this as accurate as we can.

It was my understanding that the end data for these sites exists, but has quality issues so you don't want to use it. But, maybe it's not even available because of those issues, so you have to truncate it? What's the most accurate way to describe the issues here?

Reply:
I've replaced 'bad' with 'missing', which is more accurate